src/app/Annotation.tsx

```diff
@@ -166,27 +162,27 @@ const Annotation: React.FC<AnnotationProps> = (props) => {
     case "CREATE": {
       return (
         <TranslucentBox
-          onDelete={() => {
-            dispatch({ type: "DELETE_ANNOTATION", payload: props.id });
-          }}
-          onFieldTypeChange={(value) => {
-            dispatch({
-              type: "SET_ANNOTATION_TYPE",
-              payload: { id: props.id, type: value },
-            });
-          }}
           type={type}
           nodeRef={ref}
-          css={{ cursor: "inherit", ...css }}
-        />
+          css={{ cursor: "inherit", ...css }}>
+          <FieldLayerActionMenu
```

Comment:
What do you think of the following pseudo-refactoring:

```
<FieldLayerActionMenu>
  <Button
    onClick={() => {
      dispatch({ type: "DELETE_ANNOTATION", payload: props.id });
    }}
  >
    Delete
  </Button>
  <Button
    onClick={() => {
      dispatch({
        type: "SET_ANNOTATION_TYPE",
        payload: { id: props.id, type: value },
      });
    }}
  >
    Set Annotation Type
  </Button>
</FieldLayerActionMenu>
```

Does this make sense? If we make the buttons children, this makes changing what's inside of the pop-up a little easier. Open to feedback on that, though, if you prefer passing down the props.